tests/test_opds.py

```diff
@@ -762,9 +762,10 @@ def test_page_feed(self):
         pagination = Pagination(size=1)
 
         def make_page(pagination):
+            SessionManager.refresh_materialized_views(self._db)
```

Comment:
I made these changes while diagnosing the problem--I thought it was a problem with the materialized view and that making more code use the materialized view would flush it out. Since we use the materialized view everywhere in the real app I decided it was worthwhile to keep this part of the code.